DatadogCrashReporting/Sources/Integrations/CrashReportSender.swift

```diff
@@ -66,4 +72,32 @@ internal struct MessageBusSender: CrashReportSender {
             }
         )
     }
+
+    /// Send the launch report and context to integrations.
+    ///
+    /// - Parameters:
+    ///   - launch: The launch report.
+    func send(launch: DatadogInternal.LaunchReport) {
+        guard let core = core else {
+            return
+        }
+
+        // We use baggage message to pass the launch report instead updating the global context
+        // because under the hood, some integrations start certain features based on the launch report (e.g. `WatchdogTerminationMonitor`).
+        // If we update the global context, the integrations will keep starting the features on every update which is not desired.
```

Comment:
Thanks for clarifying 👍. I'm okay with proposed way as long as there is no requirement on the order of `RUM.enable()` vs `CrashReporting.enable()` - isn't there 🤔💭?

Reply:
Keeping it for the next PR, my hunch says there will be because SDK sends the message as soon as crash reporting is enabled and if RUM is not enabled yet, it will be missed.

Which may lead to re-architecture of this.